src/facade/async.ts

```diff
@@ -130,21 +132,23 @@ export class EventEmitter<T> {
  * 
  *  ### Examples
  * 
- *  In the following esample, `ZParent` (parent component) handle `open` and `close` events emitted by `Zippy` (child component)
+ *  In the following esample, `AppComponent` (parent component) handle `open` and `close` events emitted by `Zippy` (child component)
  * 
  * ```
  * @Component({
- *   selector: 'z-parent',
- *   template: `
- *   Content is visible? {{ $ctrl.zippyState }}
- *   <zippy open="$ctrl.showState($event)" close="$ctrl.showState($event)">
- *      Alternate <i>show</i> and <i>hide</i> of this <b>content</b> ... 
- *   </zippy>
- * `})
- * export class ZParent {
- *   public zippyState: boolean = true;
- *   public showState: EventHandler<boolean> = ( visible ) => {
- *      this.zippyState = visible; // HERE visibile is a boolean === $event passed from zippy child component events
+ *   selector: 'app',
+ *   template: `<zippy open="$ctrl.onOpen($event)" close="$ctrl.onClose($event)"></zippy>`,
+ *   directives: [ZippyComponent]
+ * })
+ * export class AppComponent {
+ * 
+ *   onOpen(zippyVisible: boolean) {
+ *     // Event handler declared as simple method accepting single typed param  
+ *     console.log(`zippy visibility is: ${zippyVisible}`);
+ *   }
+ *   public onClose: EventHandler<boolean> = ( zippyVisible ) => {
+ *     // Event handler declared using explict EventHandler interface specifing the type of the param
+ *     console.log(`zippy visibility is: ${zippyVisible}`); // HERE zippyVisibile is a boolean === $event passed from zippy child component
```

Comment:
Here I modify the doc sample to show how to use the EventHandler with a function property instead of simple method, otherwise the code has no meaning becouse it doesn't show the use of interface.